flow/probes/ebpf.go

```diff
@@ -271,21 +310,31 @@ func (p *EBPFProbe) run() {
 					ebpfFlow = &EBPFFlow{
 						start: now,
 						last:  start.Add(time.Duration(lastK - startKTimeNs)),
+						Kflow: kernFlow,
 					}
 					ebpfFlows[kernFlow.key] = ebpfFlow
+
 				}
+                                if ok{
+                                    updateFromEBPF(ebpfFlow,&kernFlow)
+                                    ebpfFlow.Kflow=kernFlow
```

Comment:
seems to be useless ?

Reply:
should be the other way around after I added the above changes for we want to pass kernFlow to flowFromEBPF. Changed this as well